chicago/events.py

```diff
@@ -29,11 +30,19 @@ def eventPages(self, event_url, search_type='all') :
             return self.pages(page)
 
 
-    def scrape(self):
+    def scrape(self, follow_links=True):
         for page in self.eventPages(EVENTSPAGE):
             events_table = page.xpath("//table[@class='rgMasterTable']")[0]
             for events, headers, rows in self.parseDataTable(events_table) :
-                print(events)
+                if follow_links and type(events['Meeting\xa0Details']) == dict :
```

Comment:
perhaps we can define a string-cleanup mixin? I'd hate to make that behavior implicit (if someone's using some alternate encoding or something)